lib/nerves_hub_link/client.ex

```diff
@@ -174,6 +178,13 @@ defmodule NervesHubLink.Client do
     _ = apply_wrap(mod(), :handle_error, [data])
   end
 
+  @doc """
+  This function is called internally by NervesHubLink to notify clients of disconnects.
+  """
+  def reconnect_backoff() do
+    apply_wrap(mod(), :reconnect_backoff, [])
```

Comment:
`apply_wrap/3` has the potential to blow up and return error tuples. This would kill any reconnect trying to happen. Should we instead apply a default here as well to give it a fighting chance? Or let that error propagate to a potentially incorrect client?

Reply:
I think applying the backoff if the client returns nil is good 👍